sdarq/frontend/src/app/form/form.json

```diff
@@ -860,25 +920,30 @@
          {
-          "value": {
-           "visibleIf": "{Federal compliance oversight} = \"Interconnected with system governed by compliance\"",
-           "text": "Your application will not be hosted within a system environment that is governed by compliance such as NIST 800-53, HIPAA, HITRUST, etc. but will interconnect with this system environment. Depending on the details of the interconnection, your application may be required to align with certain information security compliance standards and/or meet specific requirements. Our compliance team will need to evaluate the interconnection to determine what, if any, requirements will be for your application. Please start a discussion with the Compliance Team."
-          },
-          "text": "Your application will not be hosted within a system environment that is governed by compliance such as NIST 800-53, HIPAA, HITRUST, etc. but will interconnect with this system environment. Depending on the details of the interconnection, your application may be required to align with certain information security compliance standards and/or meet specific requirements. Our compliance team will need to evaluate the interconnection to determine what, if any, requirements will be for your application. Please start a discussion with the Compliance Team.",
+          "value": "Your application will not be hosted within a system environment that is governed by compliance such as NIST 800-53, HIPAA, HITRUST, etc. but will interconnect with this system environment. Depending on the details of the interconnection, your application may be required to align with certain information security compliance standards and/or meet specific requirements. Our compliance team will need to evaluate the interconnection to determine what, if any, requirements will be for your application. Please start a discussion with the Compliance Team.",
           "visibleIf": "{Federal compliance oversight} = \"Interconnected with system governed by compliance\""
          },
          {
           "value": "Get Started with the SIA doc, build a concept of operations/Security Impact Assessment document. Update/build the Design Specification document. Update/build the Network Diagram and Architecture. Make a user flow of your component, make sure to show how it connects to rest of product. Make a data flow diagram of your component, make sure to show how it connects to rest of product.",
           "visibleIf": "{Federal compliance oversight} = \"System governed by compliance\""
          },
+         "Please make sure you document and limit the collection of PHI. Have documented the process of PHI if required. Allow users (owners of their PHI) to view and modify their information at will or by request.  Track all cookies used by the system and the reason for their use. Categorize cookies in accordance with GDPR and other privacy regulations. ",
```

Comment:
Seems like too much detail for one question. Maybe the specifics should be moved to a separate document listing the requirements such as cookie categories and PHI documentation. It should say what "track" means (like a document or a spreadsheet sent to compliance or a markdown file or comment in source control, whatever tracking method suffices for compliance).

If the specifics are removed to a different document, then this question can be simpler and just refer out to that document.

Reply:
Or you could just add a "Please contact" sentence as in other questions.

Reply:
Good idea, I'll ask S.Tahiri if she has a document for this.

Reply:
Fixed in recent commits. Thank you Tom!